stl/inc/memory

```diff
@@ -3610,6 +3610,7 @@ basic_ostream<_Elem, _Traits>& operator<<(basic_ostream<_Elem, _Traits>& _Out, c
 }
 
 // GARBAGE COLLECTION
+#if !_HAS_CXX23

```

Comment:
This should be controlled by a macro in yvals_core.h like the other removed features. Doing it this way does not provide an escape hatch.